.ci/docker/node-container/Dockerfile

```diff
@@ -0,0 +1,6 @@
+ARG NODE_VERSION=8
+FROM node:${NODE_VERSION}
+
+RUN npm install npm
```

Comment:
Shouldn't this be `npm install -g npm`? 🤔 

I'm not sure how containers work, but at least it seems like we have the same issues as we would have if npm was not upgraded to the latest version.

Also if you look at the actual test container it outputs that it's running with npm v3.10.10 whereas the version installed in the npm container is version 6.9.0